app.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Workarea Commerce",
+  "addons": [
+    {
+      "plan": "mongolab:sandbox",
+      "as": ["WORKAREA_MONGOID", "WORKAREA_MONGOID_METRICS"]
+    },
+    {
+      "plan": "heroku-redis:hobby-dev",
+      "as": ["WORKAREA_REDIS", "WORKAREA_REDIS_CACHE"]
+    }
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/heroku/heroku-buildpack-ruby"
+    },
+  ],
+  "env": {
```

Comment:
in review apps, the settings described here will be provided by the parent app that is already running. when deploying to heroku, a user will have to fill in this information before they can deploy (except for `HEROKU_APP_NAME` and `HEROKU_PARENT_APP_NAME`)